tailwind.config.mjs

```diff
@@ -81,12 +81,12 @@ export default {
         // This is a collection of font sizes used in existing sidebar components,
         // which will grow during the conversion-to-Tailwind process. These
         // are descriptive, not aspirational or prescriptive.
-        tiny: ['10px'],
-        sm: ['11px', '1.4'],
+        xs: ['11px', '1.4'],
+        sm: ['13px', '1.4'],
```

Comment:
What's the second value in the array for? (the `1.4`)

Reply:
`line-height`. See https://youtu.be/C_uTkUGcHv4

If you think it's a little odd (well, inconsistent) that some sizes have line-height values and others don't, you're right! Over time as I've migrated this same design across CSS implementations, if there was `line-height` set for a particular font size, I've retained it, otherwise left defaults in place. So is a sort of description of line-height decisions for the design in the past. If that makes any sense.

I am ambivalent about how Tailwind lumps in `line-height` with text size in general; it feels like a violation of their one-class, one-rule philosophy. It can be magical/invisible that line-height is getting set with a text-size rule. 🤔 Some stuff to think about here.

Reply:
Thanks for the detailed explanation!